Chinese/Hei Hei Hei/Hei Hei Hei.md

```diff
@@ -0,0 +1,14 @@
+够了！！大家都安静点！不要再扯孙悟空了，都过来看费玉污讲段子了！！
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.